javascript/ql/test/library-tests/TaintTracking/capture-flow.js

```diff
@@ -17,3 +17,15 @@ function outerMost() {
 }
 
 sink(outerMost()); // NOT OK - but missed
```

Comment:
```suggestion
sink(outerMost()); // NOT OK
```

We get this flow now.